staging/src/k8s.io/apimachinery/pkg/util/waitgroup/waitgroup.go

```diff
@@ -0,0 +1,70 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package waitgroup
+
+import (
+	"sync"
+	"sync/atomic"
+)
+
+// A WaitGroup must not be copied after first use.
```

Comment:
Needs a `WaitGroup implements the Go sync.WaitGroup semantics plus ...` above.

Reply:
I do not understand.

Reply:
It's invalid Godoc now. The convention is `WaitGroup does something.`

Reply:
OK, but golint without err. May be `a` is omitted. I will delete it